tests/server/tso/global_tso_test.go

```diff
@@ -163,12 +163,13 @@ func TestLogicalOverflow(t *testing.T) {
 			re.NoError(tsoClient.Send(req))
 			_, err = tsoClient.Recv()
 			re.NoError(err)
+			if i == 1 {
+				// the 2nd request may (but not must) overflow, as max logical interval is 262144
+				re.Less(time.Since(begin), updateInterval+20*time.Millisecond) // additional 20ms for gRPC latency
```

Comment:
Is it possible that "+20*time.Millisecond) // additional 20ms for gRPC latency" causes random failure?

Reply:
Yes, good catch.

Reply:
But it's a local environment, so the latency might not be that slow. We can solve it after we encounter it.